examples/README.md

```diff
@@ -0,0 +1,25 @@
+# Utah Design System Examples
+Here are a few examples on how to use the Utah Design System and its components.
+
+## Documentation
+
+Please see the [documentation website](https://designsystem.utah.gov/) and the [Getting Started](https://designsystem.utah.gov/resources/gettingStarted) page for all information related to the Utah Design System.
+
+## Utah Header
+We have several examples on how to implement the Design System Header:
+- [Universal Module Definition (UMD)](https://github.com/utahdts/utah-design-system/tree/dev/examples/utah-header/umd-html)
+- [ES Module](https://github.com/utahdts/utah-design-system/tree/dev/examples/utah-header/es-html)
+- [Create React App](https://github.com/utahdts/utah-design-system/tree/dev/examples/utah-header/create-react-app)
+- [Vite Plain JS App](https://github.com/utahdts/utah-design-system/tree/dev/examples/utah-header/vite)
+
+For more information on the Utah Header, please visit the [documentation page](https://designsystem.utah.gov/library/patterns/utahHeader) on the Utah Design System.
+
+
+### Site Map
+
+- [Home](https://github.com/utahdts/utah-design-system)
+- [Library](https://github.com/utahdts/utah-design-system)
+- [Design System Header](https://github.com/utahdts/utah-design-system/tree/dev/%40utahdts/utah-design-system-header)
+- Examples
```

Comment:
Is there a link for examples?

Reply:
The idea here was that, since you were on the examples README, you wouldn't need a link to it?
The "Site Map" section is really just a "README Map" of the GitHub repo. Should we rename it? Is it too confusing?

Reply:
OH! of course it doesn't need a link if it's already on that page. ha! "Sitemap" is fine.